code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -350,6 +350,39 @@ var/global/list/bottle_colour_choices = list("Blue" = "#0094FF","Dark Blue" = "#
 /obj/item/weapon/storage/pill_bottle/sweets/strange
 	items_to_spawn = list(/obj/item/weapon/reagent_containers/food/snacks/sweet/strange = 10)
 
+/obj/item/weapon/storage/pill_bottle/syndiemints
+	name = "box of mints"
+	desc = "Gets rid of halitosis and satisfied customers in one go! You shouldn't be seeing this."
+	icon = 'icons/obj/candymachine.dmi'
+	icon_state = "mintboxgeneric"
+	storage_slots = 50
+	items_to_spawn = list(/obj/item/weapon/reagent_containers/food/snacks/mint/syndiemint = 50)
+
+/obj/item/weapon/storage/pill_bottle/sweets/New()
+	..()
+	switch(3)
```

Comment:
shouldn't this be `switch(pick(3))`?

Reply:
Not sure. That's how I remember it worked, but to be fair it's been a while since I've last attempted to shitcode something.

Reply:
i havent actually ran this but i'm pretty sure it would always make `Uncle Ian's homemade mints`
you might also want to run the ..() after you edit the items_to_spawn

Reply:
Indeed you were right, I had forgotten the random value thing.

Reply:
also, put the ..() after the switch, or the thing will always contain base syndimints instead of the nano/discount/homemade ones

Reply:
true